portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#bible-verse-container {
+  margin-bottom: 20px;
+  font-style: italic;
 }
 
-#greeting-container {
+#bible-verse-text-container {
   margin-top: 20px;
 }
+
+body {
+  position: relative;
+}
+
+.content {
+  margin-left: 0;
+  margin-right: 0;
+  width: 100%;
+}
+
+.navbar {
+  transition: 0.6s;
+}
+
+.navbar .sticky-top {
+  height: 50px;
+  background-color: rgba(0, 0, 0, 0.25);
+}
```

Comment:
nit: Empty line space between code blocks, like elsewhere, before the '.section {' block.

Reply:
Done!